hnn_core/hnn_io.py

```diff
@@ -139,7 +139,19 @@ def _read_cell_response(cell_response_data, read_output):
 def _read_external_drive(net, drive_data, read_output, read_drives):
     if not read_drives:
         return None
-    if drive_data['type'] == 'evoked':
+
+    def _set_from_cell_specific(drive_data):
```

Comment:
It looks like this function is defined in another function‘s body, but it doesn’t use anything from the enclosing scope. Would it be simpler to un-nest this and put it as a separate function?